binders/kafka-binder/spring-cloud-stream-binder-kafka-streams/src/main/java/org/springframework/cloud/stream/binder/kafka/streams/InteractiveQueryService.java

```diff
@@ -298,10 +306,17 @@ public <K> KafkaStreams getKafkaStreams(String store, K key, Serializer<K> seria
 	public List<HostInfo> getAllHostsInfo(String store) {
 		return kafkaStreamsRegistry.getKafkaStreams()
 				.stream()
-				.flatMap(k -> k.allMetadataForStore(store).stream())
+				.flatMap(k -> k.streamsMetadataForStore(store).stream())
```

Comment:
How is this change related?

Reply:
It is unrelated; I just saw that the original call was deprecated and decided to clean up inline.

Reply:
I would leave it as-is for this PR. The reason being is that it is deprecated in kafka-streams 3.3.1 but I think there is slight behavior change in the 2 methods across earlier versions of kafka-streams.